plugin.video.lacapitv/resources/lib/scraper.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# KodiAddon (La Capi TV)
+#
+from t1mlib import t1mAddon
+import json
+import re
+import xbmcplugin, xbmcaddon
+import xbmcgui
+import sys
+import xbmc
+import requests
+
+settings = xbmcaddon.Addon(id="plugin.video.lacapitv")
+cuenta = ""
+path = ""
+if settings.getSetting('activa') == "true":
+	path ="p/"
+	cuenta = settings.getSetting('username') + ":" + settings.getSetting('password') + "@"
+html = requests.get('https://'+cuenta+'play.lacapi.tv/'+path+'show.json').text
```

Comment:
bad practice to store the html in the scope of the script. Can't you get it only when needed? Eg in getAddonMenu before json.loads?

Reply:
That doesn't work well with my server's header. And it creates problems.